### DISCUSSION TITLE
Headless service can't comunicate after installed

### DISCUSSION THREAD

**Discussion by kittisuw** [linkerd/linkerd2#9233] [Q&A]
version: stable-2.11.4
As I read https://linkerd.io/2018/11/14/grpc-load-balancing-on-kubernetes-without-tears.
Normally linkerd has support both headless or non-headless. But after I installed and inject deployments using that using grpc with headless service. All services can't communicate. How does it needs more action after installed?

**Comment by kleimkuhler** [comment]
Hey @kittisuw we'll need a little more information about what is happening and why your services can't communicate. Steps for a reproduction would be helpful as well as logs to get a better ideas about what errors are actually occurring.
